performant-software/arc-visualization
# made filters look like buttons and added touch functionality

### What does this PR do?
- Made Visualize By filters look more like buttons and changed the active color of the button to match the color of the visualization bubble type. 
- Added touch functionality to use more fluidly on a touch screen device. 
- Issues with pinch zoom on touch screen still exist, so for time being, until those issues can be resolved, added buttons for zoom in and zoom out to help with use on touch devices. 

### What issues does it address?
- Closes #1 and Closes #5 

### How to test:
- Access the BigDiva page by going to 192.237.180.236
- Take note that directly under the BigDIVA logo in the upper left, there are buttons to select different filters by which to visualize by. 
- Notice that the buttons, when selected, match the color of the bubbles currently being displayed. 
- Note that the buttons, when selected, deselect any other button that may have been in use. 
- Try out all different actions on a touch screen device to verify that those actions work via touch input. (Note, two finger pinch zoom in and zoom out are currently unsupported. To zoom in or out using touch, use the Zoom In and Zoom Out buttons.)
